src/lavinmq/http/controller/view_helpers.cr

```diff
@@ -14,6 +15,10 @@ module LavinMQ
         HTML.escape({{value}}, context.response)
       end
 
+      macro active_path?(path)
```

Comment:
Why would this be a macro and not just a normal method?

Reply:
To not have to know about `route_path` and pass it as an argument too.

Reply:
ok, i get it that macro might be needed here, but can `route_path` be replaced with `context.request.path` or something like that? `context` feels more constant than the `route_path` variable at least.

Reply:
replaced it with 

```
context.request.path == "/#{{{path}}}" || (context.request.path == "/" && {{path}} == :".")
```

because then we could drop the `route_path` variable, and no need for `strip`, which would allocate a new string. the interoplation `"/#{{{path}}"` allocates a new string, but so does `{{path}}.to_s`, which we can't get away with.